control-plane/agents/src/bin/core/controller/reconciler/pool/mod.rs

```diff
@@ -141,8 +141,8 @@ async fn missing_pool_state_reconciler(
         async {
             pool_spec.warn_span(|| tracing::warn!("Attempting to recreate missing pool"));
 
-            let request = CreatePool::new(&pool_spec.node, &pool_spec.id, &pool_spec.disks, &pool_spec.labels);
-            match node.create_pool(&request).await {
+            let request = ImportPool::new(&pool_spec.node, &pool_spec.id, &pool_spec.disks);
```

Comment:
How do we check the disks have not been swapped? Also what happens if the disk is correct and the pool is missing?

Reply:
Then import will fail.